health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/DrawerFieldComposer.js

```
@@ -0,0 +1,189 @@
+          <Button
+            type={"button"}
+            size={"small"}
+            variation={"teritiary"}
+            label={t("ADD_OPTIONS_FOR_DROPDOWN_APP")}
+            onClick={
+              () =>
+                setDrawerState((prev) => ({
+                  ...prev,
+                  dropDownOptions: prev?.dropDownOptions
+                    ? [
+                        ...prev?.dropDownOptions,
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)